src/pretix/base/settings.py

```diff
@@ -464,7 +480,49 @@ def unserialize(cls, s):
             label=_('Only end payment terms on weekdays'),
             help_text=_("If this is activated and the payment term of any order ends on a Saturday or Sunday, it will be "
                         "moved to the next Monday instead. This is required in some countries by civil law. This will "
-                        "not effect the last date of payments configured above."),
+                        "not effect the last date of payments configured below."),
+            widget=forms.CheckboxInput(
+                attrs={
+                    'data-display-dependency': '#id_payment_term_mode_0',
+                    'data-required-if': '#id_payment_term_mode_0'
```

Comment:
@felixrindt I'm curious. Why do we need to force events that have `payment_term_mode` set to `days` to only end payment terms on weekdays? What if this is not required by the law in the country where the event is located? Is this a feature or a bug?

Reply:
This indeed seems like an oversight. We only need it to be displayed, but not checked, when `days` are selected. Glad you noticed it, thanks! I created #1862 to fix it.